src/config/webpack.dev.config.babel.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable max-len */
+
+import path from 'path';
+import webpack from 'webpack';
+
+import webpackConfig from './webpack.prod.config.babel';
+import config from './index';
+
+import WebpackIsomorphicToolsPlugin from 'webpack-isomorphic-tools/plugin';
+import webpackIsomorphicToolsConfig from './webpack-isomorphic-tools';
+
+const webpackIsomorphicToolsPlugin =
+  new WebpackIsomorphicToolsPlugin(webpackIsomorphicToolsConfig);
+const APP_NAME = config.get('currentApp');
+const BABEL_QUERY = {
```

Comment:
And there's a reason you can't do it directly that way because I started with that but I can't remember why there's a reason you can't do it via env in .babelrc.

In any case if we remove the transform stuff for HMR should be superfluous.